### ISSUE TITLE
tests fail (test/test executable)

### ISSUE THREAD

**Issue by yurivict** [Gecode/gecode#109]
**Describe the bug**
```
===>  Testing for gecode-6.3.0
AFC +++++
Array::Iterator::SharedArray +++++
Array::Iterator::VarArgs +++++
Array::Iterator::VarArray +++++
Array::Iterator::ViewArray +++++
Bool::Assign::3 +++++
Bool::Assign::5 +++++
Bool::Assign::7 +++++
Bool::Branch::3 +++++
Bool::Branch::5 +++++
Bool::Branch::7 +++++
FlatZinc::2DPacking +++++
FlatZinc::SharedArrayElement -
Options: -seed 812278110 -test FlatZinc::SharedArrayElement
*** Error code 1
```

6.3.0 is from the corresponding branch
OS: FreeBSD 12.2
clang-10

**Comment by zayenz** [comment]
Yes, I've seen this as well and I am looking into it.

**Comment by zayenz** [comment]
Fixed in release/6.3.0 (ccede61) and develop (bbefcea).
